.travis.yml

```diff
@@ -12,5 +12,5 @@ install:
 
 script:
   - gb build
-  - golint src/
+  - golint -set_exit_status src/*
```

Comment:
From golint docs, `golint [flags] [directories] # where a '/...' suffix includes all sub-directories`

Reply:
I see, both worked! I'll change it